feat: CI to publish helm chart

ts-mini opened the pull request:
dependent on #7 - just an opinionated approach to publishing helm charts, but using github first this time

pvcnt commented:
Hi! I tested this workflow file and came up with some changes. I removed helm-docs (I don't think it was useful since this was not pushed anywhere actually), and replaced the helm releaser by a plain and simpler `helm package` call. It is also only pushing to ghcr.io (not both gh-pages and ghcr.io). It is installed with the following command:

```bash
helm upgrade --install reviewer oci://ghcr.io/pvcnt/helm/reviewer --version ${version}
```

I validated this in a test branch and its does seem to work. Since I pushed changes to your branch, please let me know what you think of it! Does it fit your use case?

ts-mini commented:
> Hi! I tested this workflow file and came up with some changes. I removed helm-docs (I didn't get how it was used?), and replaced the helm releaser by a plain and simpler `helm package` call. It is also only pushing to ghcr.io. It is installed with the following command:
> 
> ```shell
> helm upgrade --install reviewer oci://ghcr.io/pvcnt/helm/reviewer --version ${version}
> ```
> 
> I validated this in a test branch and its does seem to work. Since I pushed changes to your branch, please let me know what you think of it! Does it fit your use case? Do you also need the publication of a relase on GitHub + publication on gh-pages, like what the chart-releaser-action was doing? I'd like to understand your workflow, and where would you like to install the Helm chart from.

honestly I keep forgetting you can do oci images - there this is totally fine.  The use case was "i dont want to clone and install from local dir" and this completely solves that.


sorry for the late reply!

pvcnt commented:
Let's merge this, then. Thank you!
